powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/request/OperationApproveInternalRequest.java

```diff
@@ -31,6 +32,7 @@ public class OperationApproveInternalRequest {
     private String activationId;
     @NotBlank
     private String applicationId;
+    private String signatureType;
```

Comment:
My idea was to create a type-safe parameter and introduce a `com.wultra.security.powerauth.app.testserver.model.enumeration.SignatureType`.

Reply:
This is very cool, but then could we fix this everywhere?